---
layout: archive
title: "GitHub"
permalink: /github/
classes: wide
---

*Links to GitHub repositories for recent studies:*

[Rattlesnake venom population genomics](https://github.com/drewschield/venom_population_genomics)<a href="#" class="btn btn-info">Success Button</a>

We analyzed population genomic data from venom gene regions in rattlesnake populations to characterize genetic diversity and signatures of selection.

[Male-biased mutation rates in rattlesnakes](https://github.com/drewschield/male-biased_mutation_crotalus){: .btn .btn--info .text-center}

We compared levels of autosomal and Z-linked sequence divergence between rattlesnake (*Crotalus*) species to infer a ratio of male-to-female mutation rate. We found evidence of a ~2-fold male-biased mutation rate, which has ramifications for rates of evolution in sex-linked genes in snakes.

[Z-linked and autosomal genomic variation in barn swallows](https://github.com/drewschield/Z-chromosome_analysis_hirundo){: .btn .btn--info .text-center}

Using a whole-genome resequencing dataset, we explored genetic diversity and differentiation across the genome between barn swallow (*Hirundo rustica*) subspecies. We compared variation on the Z chromosomes and autosomes to understand how different evolutionary processes have impacted sex-linked genetic diversity in this young species complex. We found evidence that low sex-linked diversity has been shaped by a combination of excess variance in male reproductive success, recent population bottlenecks, linked selection, and reduced gene flow on the Z chromosome.

[Snake recombination](https://github.com/drewschield/recombination){: .btn .btn--info .text-center}

We analyzed fine-scale LD-based recombination maps from rattlesnake species to characterize the genomic recombination landscape, annotate recombination hotspots, identify associations between hotspots and genomic features, and test the hypothesis that PRDM9 directs recombination hotspots in snake genomes.