---
layout: archive
permalink: /research/
title: "Research"
classes: wide
author_profile: true
---

### Genetics of Speciation and Adaptation

My research is motivated by interests in the formation of species and how they adapt to their environments, and I use comparative and population genomic datasets to investigate these processes. I often use hybrid zones between historically isolated populations to study the interplay between forms of selection and gene flow and to understand how certain genomic regions (and the traits they underlie) affect reproductive isolation. Much of this work has focused on rattlesnake species ([Schield et al. 2015 *MPE*](https://drewschield.github.io/pubs/2015SchieldMPE.pdf), [2017 *Ecol & Evol*](https://drewschield.github.io/pubs/2017SchieldEcolEvol.pdf), [2018 *MPE*](https://drewschield.github.io/pubs/Schield2018MPEv2.pdf), [Schield and Perry et al. 2019 *Biol J Linn Soc*](https://drewschield.github.io/pubs/SchieldPerry2019BiolJLinnSoc.pdf), [Nikolakis et al. 2022 *Evolution*](https://drewschield.github.io/pubs/Nikolakis2022Evolution.pdf)), in which there are many cases of natural hybridization between populations with amazing variation in adaptive traits. I also study speciation and the genetic architecture of sexually selected traits in barn swallows using hybrid zones ([Schield et al. 2021 *Mol Ecol*](https://drewschield.github.io/pubs/Schield2021MolEcol.pdf), [Turbek et al. 2022 *Evolution*](https://drewschield.github.io/pubs/Turbek2022Evolution.pdf)), with ongoing work to test the hypothesis that selection for specific combinations of traits can promote strong barriers to gene flow.

Other work focuses on how different processes shape adaptive evolution. Snake venom is an adaptive trait of considerable interest, yet relatively little work has taken advantage of population genetics to understand how different processes contribute to its evolution, especially in the context of antagonistic coevolution with mechanisms of prey resistance. We recently used genomes from rattlesnake populations to quantify population genetic variation in venom gene regions, finding evidence for a prominent role of long-term balancing selection in maintaining adaptive genetic variation ([Schield et al. 2022 *Nature Ecol Evol*]( https://drewschield.github.io/pubs/Schield2022NatureEE.pdf)). Future research in this system will consider the population genetics of antagonistic coevolution, both from the perspective of snake venom and resistance mechanisms in prey species.

### Evolutionary Genomics

Genome evolution is fascinating, especially in the context of adaptation and speciation. My research approaches this from a variety of angles, ranging from from the structure and function of individual genomes, to comparative genomics among species, and population genomics to investigate how different evolutionary forces shape genetic diversity. I've contributed to several projects investigating genome structure and chromosome evolution in vertebrates, especially snakes ([Castoe et al. 2013 *PNAS*](https://drewschield.github.io/pubs/2013CastoePNAS.pdf), [Pasquesi et al. 2018 *Nat Commun*](https://drewschield.github.io/pubs/Pasquesi2018NatureCommun.pdf), [Perry and Card et al. 2018 *GBE*](https://drewschield.github.io/pubs/Perry2018GBE.pdf)), and genomic mechanisms underlying adaptive traits in snakes, such as venom ([Schield et al. 2019 *Genome Res*](https://drewschield.github.io/pubs/Schield2019GenomeRes.pdf), [Perry et al. 2022 *Genome Res*](https://drewschield.github.io/pubs/Perry2022GenomeRes.pdf), [Gopalan et al. 2022 *Toxicon*](https://drewschield.github.io/pubs/Gopalan2022Toxicon.pdf)) and physiological organ regeneration upon feeding ([Andrew and Perry et al. 2017 *BMC Genomics*](https://drewschield.github.io/pubs/2017AndrewBMCGenomics.pdf), [Perry et al. 2019 *Proc B*](https://drewschield.github.io/pubs/Perry2019ProcB.pdf)).<br>

Of particular interest is how genetic recombination shapes the processes of adaptation and speciation and the factors that contribute to variation in recombination rate across the genome. Previously, I studied recombination rate variation in rattlesnake species ([Schield et al. 2020 *MBE*](https://drewschield.github.io/pubs/Schield2020_MBE_format.pdf)), finding a combination of patterns somewhat unique among vertebrates, and which indicates that snakes and other reptiles are important lineages to investigate to more thoroughly understand meiotic recombination in vertebrates. Ongoing work will explore drivers of intra- and interspecific recombination rate variation in snakes and the interaction between recombination and other processes in adaptation and speciation.

### Sex Chromosome Evolution

<!-- <img style="float: left;" src="{{ https://drewschield.github.io }}{{ https://drewschield.github.io }}/images/sexchrom_exp.png" alt="" width="300"/> -->
Sex chromosomes are intriguing regions of the genome due to their unique patterns of inheritance, recombination suppression and degeneration, and relevance to speciation and adaptation. I am especially interested in sex chromosome evolution in snakes and birds, and have contributed to work showing independent origins of XY and ZW systems in snakes ([Gamble et al. 2017 *Current Biology*](https://drewschield.github.io/pubs/Gamble2017CurrentBiol.pdf)), the evolution of partial dosage compensation in snakes, and evidence for a recent and largely undegenerated evolutionary stratum on the rattlesnake W Chromosome ([Schield et al. 2019 *Genome Res*](https://drewschield.github.io/pubs/Schield2019GenomeRes.pdf)). I'm currently working on other projects focused on sex chromosome evolution in ZW snakes.<br>

### Phylogenetics and Phylogeography

<!-- ![image-right]({{ https://drewschield.github.io }}{{ https://drewschield.github.io }}/images/scut_tree.png =400x){: .align-right} -->
<!-- <img style="float: right;" src="{{ https://drewschield.github.io }}{{ https://drewschield.github.io }}/images/scut_tree.png" alt="" width="400"/> -->
Understanding the evolutionary history of a group is intrinsically valuable and provides a necessary framework for genomic comparisons. I've contributed to several projects focused on estimating the phylogeny and phylogeography of snake species, with a special focus on comparisons of nuclear and mitochondrial trees. Differences in topologies between nuclear and mitochondrial genomes can reveal interesting patterns, including differential introgression in hybrid zones ([Schield et al. 2015 *MPE*](https://drewschield.github.io/pubs/2015SchieldMPE.pdf), [2017 *Ecol & Evol*](https://drewschield.github.io/pubs/2017SchieldEcolEvol.pdf), [Schield and Perry et al. 2019 *Biol J Linn Soc*](https://drewschield.github.io/pubs/SchieldPerry2019BiolJLinnSoc.pdf)), and distinct patterns of geographic range expansion (i.e., genetic surfing; [Streicher et al. 2016 *Evolution*](https://drewschield.github.io/pubs/2016StreicherEvolution.pdf)).
