---
layout: archive
permalink: /research/
title: "Research"
classes: wide
author_profile: true
---

### Speciation and Hybridization

My research is motivated by an interest in the the formation of species, and I use comparative and population genomic datasets and to investigate this process. I am also interested in using hybrid zones between historically isolated populations to quantify patterns of genomic introgression, ultimately to understand how divergent genomic backgrounds admix in scenarios of secondary contact with gene flow, and how reproductive isolation may take the form of genomic regions that resist introgression due to selection. My dissertation work focused on population structure, gene flow, demography, and genomic introgression in rattlesnake species ([Schield et al. 2015 *MPE*](https://drewschield.github.io/pubs/2015SchieldMPE.pdf), [2017 *Ecol & Evol*](https://drewschield.github.io/pubs/2017SchieldEcolEvol.pdf), [2018 *MPE*](https://drewschield.github.io/pubs/Schield2018MPEv2.pdf), [Schield and Perry et al. 2019 *Biol J Linn Soc*](https://drewschield.github.io/pubs/SchieldPerry2019BiolJLinnSoc.pdf)) with my PhD advisor [Todd Castoe](http://www.castoelaboratory.org/people/toddcastoe/). I am currently studying speciation in Barn Swallows using samples collected across hybrid zone transects, in collaboration with [Becca Safran](http://www.safran-lab.com/current-members.html).<br>

### Evolutionary Genomics

Genome evolution is fascinating, especially in the context of adaptation and speciation. My research approaches this from a variety of angles, ranging from from the structure and function of individual genomes, to comparative genomics among species, and population genomics to investigate how different evolutionary forces shape genetic diversity. I've contributed to several projects investigating genome structure and chromosome evolution in vertebrates, especially snakes ([Castoe et al. 2013 *PNAS*](https://drewschield.github.io/pubs/2013CastoePNAS.pdf), [Pasquesi et al. 2018 *Nat Commun*](https://drewschield.github.io/pubs/Pasquesi2018NatureCommun.pdf), [Perry and Card et al. 2018 *GBE*](https://drewschield.github.io/pubs/Perry2018GBE.pdf)), and genomic mechanisms underlying adaptive traits in snakes, such as venom ([Schield et al. 2019 *Genome Res*](https://drewschield.github.io/pubs/Schield2019GenomeRes.pdf)) and physiological organ regeneration upon feeding ([Andrew and Perry et al. 2017 *BMC Genomics*](https://drewschield.github.io/pubs/2017AndrewBMCGenomics.pdf), [Perry et al. 2019 *Proc B*](https://drewschield.github.io/pubs/Perry2019ProcB.pdf)).<br>

More recently, I studied recombination rate variation across the genome of rattlesnake species ([Schield et al. 2020 *MBE*](https://drewschield.github.io/pubs/Schield2020_MBE_format.pdf)). We found a combination of patterns that is somewhat unique among vertebrates, and which indicates that snakes and other reptiles could an important lineage to investigate to more thoroughly understand meiotic recombination in vertebrates.<br>

### Sex Chromosome Evolution

<!-- <img style="float: left;" src="{{ https://drewschield.github.io }}{{ https://drewschield.github.io }}/images/sexchrom_exp.png" alt="" width="300"/> -->
Sex chromosomes are intriguing regions of the genome due to their unique patterns of inheritance, recombination suppression and degeneration, and relevance to speciation and adaptation. I am especially interested in sex chromosome evolution in snakes and birds, and have contributed to work showing independent origins of XY and ZW systems in snakes ([Gamble et al. 2017 *Current Biology*](https://drewschield.github.io/pubs/Gamble2017CurrentBiol.pdf)), the evolution of partial dosage compensation in snakes, and evidence for a recent and largely undegenerated evolutionary stratum on the rattlesnake W Chromosome ([Schield et al. 2019 *Genome Res*](https://drewschield.github.io/pubs/Schield2019GenomeRes.pdf)). I'm currently working on other projects focused on sex chromosome evolution in ZW snakes.<br>

### Phylogenetics and Phylogeography

<!-- ![image-right]({{ https://drewschield.github.io }}{{ https://drewschield.github.io }}/images/scut_tree.png =400x){: .align-right} -->
<!-- <img style="float: right;" src="{{ https://drewschield.github.io }}{{ https://drewschield.github.io }}/images/scut_tree.png" alt="" width="400"/> -->
Understanding the evolutionary history of a group is intrinsically valuable and provides a necessary framework for genomic comparisons. I've contributed to several projects focused on estimating the phylogeny and phylogeography of snake species, with a special focus on comparisons of nuclear and mitochondrial trees. Differences in topologies between nuclear and mitochondrial genomes have can reveal interesting patterns, including differential introgression in hybrid zones ([Schield et al. 2015 *MPE*](https://drewschield.github.io/pubs/2015SchieldMPE.pdf), [2017 *Ecol & Evol*](https://drewschield.github.io/pubs/2017SchieldEcolEvol.pdf), [Schield and Perry et al. 2019 *Biol J Linn Soc*](https://drewschield.github.io/pubs/SchieldPerry2019BiolJLinnSoc.pdf)), and distinct patterns of geographic range expansion (i.e., genetic surfing; [Streicher et al. 2016 *Evolution*](https://drewschield.github.io/pubs/2016StreicherEvolution.pdf)).
