---
layout: splash
header:
  title: "test"
  overlay_color: "#000"
  overlay_filter: "0.5"
  overlay_image: /images/crater.jpg
  cta_label: "About"
  cta_url: "/about/"
excerpt: "Population genomics of speciation and adaptation."
feature_row:
  - image_path: /images/Fig1MBE.jpg
    alt: "Snake recombination landscapes"
    title: "Featured Publication"
    excerpt: 'Snake recombination landscapes are concentrated in functional regions despite PRDM9. 2020. *__Molecular Biology and Evolution__* 37: 1272-1294.<br><br>Meiotic recombination in vertebrates is concentrated in hotspots throughout the genome. The location and stability of hotspots have been linked to the presence or absence of PRDM9, leading to two primary models for hotspot evolution derived from mammals and birds. Species with PRDM9-directed recombination have rapid turnover of hotspots concentrated in intergenic regions (i.e., mammals), while hotspots in species lacking PRDM9 are concentrated in functional regions and have greater stability over time (i.e., birds). Snakes possess PRDM9, yet virtually nothing is known about snake recombination. Here we examine the recombination landscape and test hypotheses about the roles of PRDM9 in rattlesnakes. We find substantial variation in recombination rate within and among snake chromosomes, and positive correlations between recombination rate and gene density, GC content, and genetic diversity. Like mammals, snakes appear to have a functional and active PRDM9, but rather than being directed away from genes, snake hotspots are concentrated in promoters and functional regions – a pattern previously associated only with species that lack a functional PRDM9. Snakes therefore provide a unique example of recombination landscapes in which PRDM9 is functional, yet recombination hotspots are associated with functional genic regions – a combination of features that defy existing paradigms for recombination landscapes in vertebrates. Our findings also provide evidence that high recombination rates are a shared feature of vertebrate microchromosomes. Our results challenge previous assumptions about the adaptive role of PRDM9 and highlight the diversity of recombination landscape features among vertebrate lineages.'
    url: "https://drewschield.github.io/pubs/Schield2020_MBE_format.pdf"
    btn_label: "Read"
    btn_class: "btn--success"
---

{% include feature_row id="feature_row" type="left" %}
